proxmoxtf/resource/vm.go

```diff
@@ -4798,6 +4842,10 @@ func vmUpdate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.D
 		return diag.FromErr(e)
 	}
 
+	if vmUpdatePool(ctx, d, api.Pool(), vmID) != nil {
+		return diag.FromErr(e)
```

Comment:
this swallows the returned error value and uses error `e` from the previous statement

Reply:
Woops, sorry. Thanks for the fix.